disnake/appinfo.py

```diff
@@ -402,6 +448,154 @@ def user_install_type_config(self) -> Optional[InstallTypeConfiguration]:
         """
         return self._install_types_config.get(1)
 
+    async def edit(
+        self,
+        *,
+        custom_install_url: Optional[str] = MISSING,
+        description: Optional[str] = MISSING,
+        role_connections_verification_url: Optional[str] = MISSING,
+        install_params: Optional[InstallParams] = MISSING,
+        guild_install_type_config: Optional[InstallTypeConfiguration] = MISSING,
+        user_install_type_config: Optional[InstallTypeConfiguration] = MISSING,
+        flags: ApplicationFlags = MISSING,
+        icon: Optional[AssetBytes] = MISSING,
+        cover_image: Optional[AssetBytes] = MISSING,
+        interactions_endpoint_url: Optional[str] = MISSING,
+        tags: Sequence[str] = MISSING,
+        event_webhooks_url: Optional[str] = MISSING,
+        event_webhooks_status: ApplicationEventWebhookStatus = MISSING,
+        event_webhooks_types: Sequence[str] = MISSING,
+    ) -> AppInfo:
+        """|coro|
+
+        Edit's the application's information.
+
+        All parameters are optional.
+
+        .. versionadded:: |vnext|
+
+        Parameters
```

Comment:
very pedantic, but it would be nice if the order of these matched the parameters/api docs